website/project/views/node.py

```diff
@@ -1068,7 +1070,11 @@ def search_node(**kwargs):
     auth = kwargs['auth']
     node = Node.load(request.json.get('nodeId'))
     include_public = request.json.get('includePublic')
+    size = float(request.json.get('size', '5').strip())
```

Comment:
this is set to float for the later math calculate https://github.com/CenterForOpenScience/osf.io/pull/2075/files#diff-033d4b3df3102b7d1aace074938585e9R1100 
pages = math.ceil(count / size)